tests/integ/test_session.py

```diff
@@ -681,6 +682,31 @@ def test_cte_optimization_enabled_on_session(db_parameters):
         assert new_session2.cte_optimization_enabled is True
 
 
+@pytest.mark.skipif(IS_IN_STORED_PROC, reason="Can't create a session in SP")
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
```

Comment:
This should be an xfail instead. We distinguish between xfail and skipif such that xfail is for tests that will not be supported in Local Testing and skipif is for tests that are blocked on feature or bug fix in development.

Reply:
updated to xfail, i think I was copying this from other similar tests.